TechDraw/v018v019Migration.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
```

Comment:
Can you please rename with a name starting with `TechDraw` with the extension `FCMacro`, e.g. TechDrawMigrateFromV018.FCMacro`.